.controlplane/entrypoint.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash -e
```

Comment:
why the `-e` flag?

Reply:
I applied it based on the new Rails 7.1 Dockerfile. It is needed for the next changes to run; the `==` operation in the condition.

It is to make the whole Dockerfiles have similar content, making it easier to debug issues in future.